pkg/components/components.go

```diff
@@ -742,6 +742,9 @@ func GetClusterRole(allowMultus bool) *rbacv1.ClusterRole {
 					"get",
 					"list",
 					"watch",
+					"create",
```

Comment:
one thing we can consider
limit this to specific NS (i.e default atm)
or use it as Role instead RoleBinding with limited NS which is the default NS

Reply:
My thoughts exactly.. we can decide to use more specifc RBAC when we decide what the final resting place of this NAD